.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: false
+
+language: go
+
+go:
+  - 1.11.x
+  - master
+
+env:
+  - GO111MODULE=off
+  - GO111MODULE=on
+
+matrix:
+  allow_failures:
+    - go: master
+
+go_import_path: github.com/prometheus/node_exporter
```

Comment:
Nope, that's correct. We do not change import paths in code. That line tells Travis CI to close sources to `$GOPATH/github.com/prometheus/node_exporter` where it can be compiled without modules.